nexus/external-api/src/lib.rs

```diff
@@ -26,7 +26,7 @@ use omicron_common::api::external::{
 use openapi_manager_types::ValidationContext;
 use openapiv3::OpenAPI;
 
-pub const API_VERSION: &str = "20250409.0.0";
+pub const API_VERSION: &str = "20250604.0.0";
```

Comment:
It looks to me like this constant only exists to be checked against what's in the OpenAPI document in the test in this file.  Is that right?

Reply:
I don't really have context, I've just been blindly bumping it, but yes?